events.md

```diff
@@ -7,4 +7,4 @@ navigation_order: 1
 
 These are the planned events for Monthly Music Hackathon. Check out each one for more details and to RSVP.
 
-For past events, check in the [Event Archive](/events/archive.html).
```

Comment:
Check the _site/ folder locally to see what actually gets output from the build process.